src/objects/DataImportBatch__c.object

```diff
@@ -360,6 +360,7 @@
         <columns>NAME</columns>
         <columns>LAST_UPDATE</columns>
         <columns>Batch_Description__c</columns>
+        <columns>GiftBatch__c</columns>
```

Comment:
Is this adding a field to the DataImportBatch object and do we want to do that?

Reply:
The field was added previously for BGE. This adds it to the "All" list view of DataImportBatch. In the TSO @jesslopez-sf did that as a customization to the packaged list view, but I thought it would be simpler to make the change in the packaged list view rather than duplicate the list view definition with a different list of columns in unpackaged/trial. Is there a reason not to make this change in the package?